### DISCUSSION TITLE
Find a simple way of integrating options frameworks for themes

### DISCUSSION THREAD

**Discussion by robruiz** [wprig/wprig#694] [Ideas]
The idea here is to make it easier for people to add custom options to their theme. Whether that be using the Customizer API to do it through that built-in WP tool, or to use more robust 3rd party options frameworks like Redux Framework. 

Perhaps this could be achieved via a CLI command where it just grabs the repo, puts it in the proper place in the theme, then generates the necessary config file to get it all wired up. Then all the theme dev would need to do is edit an associated array to get their desired fields all set up.

**Comment by LukaszJaro** [comment]
> For the users who take the next step, widgets and nav menus — at least the traditional admin screens — will disappear. The customizer, which was once touted as the future of theme development, is also getting the ax. Site customization via a system where everything is a block will reign supreme.

-https://wptavern.com/what-is-full-site-editing-and-what-does-it-mean-for-the-future-of-wordpress
